browser-test/src/support/applicant_questions.ts

```diff
@@ -741,6 +742,52 @@ export class ApplicantQuestions {
     }
   }
 
+  async expectCommonIntakeConfirmationPageNorthStar(
+    wantUpsell: boolean,
+    wantTrustedIntermediary: boolean,
+    wantEligiblePrograms: string[],
+    northStarEnabled = true,
+  ) {
+    if (wantTrustedIntermediary) {
+      await expect(
+        this.page.getByRole('heading', {
+          name: 'Programs your client may qualify for',
+        }),
+      ).toBeVisible()
+    } else {
+      await expect(
+        this.page.getByRole('heading', {name: 'Programs you may qualify for'}),
+      ).toBeVisible()
+    }
+
+    if (wantUpsell) {
+      await expect(
+        this.page.getByRole('heading', {
+          name: 'Create an account to save your application information',
+        }),
+      ).toBeVisible()
+    } else {
+      await expect(
+        this.page.getByRole('heading', {
+          name: 'Create an account to save your application information',
+        }),
+      ).toBeHidden()
+    }
+
+    // Prior to North Star, the program name was rendered with
```

Comment:
**praise:** Good call out / question! We can update the class in the template and update the locator here to something more descriptive: https://github.com/civiform/civiform/blob/35bbff23f6f9d6729fb045c454b24e0b814d219a/server/app/views/applicant/ApplicantUpsellTemplate.html. I imagine it was just cause there wasn't a test added for this yet that nobody added a more helpful class name.

Reply:
Acknowledged. Would you like me to make that change now, or can that be saved for a future PR?

Reply:
Feel free to do it in this PR or in a fast follow. But if we save it for a follow up PR, could you update this comment to something like `TODO(#___): rename class to be more descriptive` with either the ticket number of this migration bug or another issue to track this?